src/editors/xrSdkControls/Controls/TreeViewSearchPanel/TreeViewSearchPanel.cs

```diff
@@ -72,7 +72,7 @@ private void searchBox_KeyDown(object sender, KeyEventArgs e)
                     MessageBox.Show("There is no matches!");
                 else if (findedNodes.Count == 1)
                     resultLabel.Text = "Result: 1 / 1";
-                else if (findedNodes.Count > 1)
```

Comment:
And now with have two `else { ... } else { ... }` branches

Reply:
Sorry, output window has hidden last else from me. Just removed dead code.